### PR TITLE
Feature/admin workload points screen

### PR DESCRIPTION
**Pull request by caoimheb**

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -91,6 +91,104 @@ router.get('/admin', (req, res) => {
...
+    cusd2: 25,
+    cusd1: 13,
+    cusc2: 23,
+    cusc1: 24,
+    cusb2: 45, 
+    cusb1: 46,
+    cusa: 58
+  }
+
+  res.render('admin/workload-points/workload-points.html', {
```
Comment (tsheppard01) [inline app/routes.js]: Don't think you need to .html?

File: app/views/admin/workload-points/workload-points.html [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+  <div class="js-tabs">
+		<ul class="js-tabs-links">
+			<li><a href="#custody">Custody</a></li>
+			<li><a href="#license">License</a></li>
+			<li><a href="#community">Community</a></li>
+			<li><a href="#other">Other</a></li>
+		</ul>
+
+		<div class="js-tabs-content">
```
Comment (tsheppard01) [inline app/views/admin/workload-points/workload-points.html:26]: Indentation is off here

File: app/views/admin/workload-points/workload-points.html [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+                  <th scope="row">Lic A</th>
+                  <td class="numeric">5</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+      
+      <div id="community">
```
Comment (tsheppard01) [inline app/views/admin/workload-points/workload-points.html:150]: Would it be cleaner if each of these tables was an includes?

File: app/views/admin/workload-points/workload-points.html [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+      <strong class="bold-small">
+        Saving changes made here will cause reports to be updated. The report update process can take 30 minutes to complete, in which time no further changes can be saved.
+      </strong>
+    </div>
+    <br>  
+    <div class="form-group">
+      <input class="button" type="submit" value="Save">
+    </div>
+  {% else %}
+    <a href="/admin/workload-points/edit" class="button">Edit</a>
```
Comment (tsheppard01) [inline app/views/admin/workload-points/workload-points.html:290]: This edit button appears at the bottom of every table.  It brings back to the custody tab

File: app/routes.js (hunk 1 above)
Comment (caoimheb) [reply app/routes.js]: Indeed I don't. Removed

File: app/views/admin/workload-points/workload-points.html (hunk 2 above)
Comment (caoimheb) [reply app/views/admin/workload-points/workload-points.html:26]: Is it? How?

File: app/views/admin/workload-points/workload-points.html (hunk 4 above)
Comment (caoimheb) [reply app/views/admin/workload-points/workload-points.html:290]: Aye, only the custody tab actual functions. The rest are just for show :)

File: app/views/admin/workload-points/workload-points.html (hunk 2 above)
Comment (tsheppard01) [reply app/views/admin/workload-points/workload-points.html:26]: Hohum, looks weird in Github, but not in VSC

File: app/views/admin/workload-points/workload-points.html (hunk 3 above)
Comment (caoimheb) [reply app/views/admin/workload-points/workload-points.html:150]: Maybe in the real app, but not sure it's worth it for the prototype. We don't apply the same structure in our app anyway

**Comment by tsheppard01** [comment]
LGTM

**Review by tsheppard01 — APPROVED** [review]

**Pull request merged**